modules/backend/formwidgets/CodeEditor.php

```diff
@@ -102,6 +102,11 @@ class CodeEditor extends FormWidgetBase
      */
     public $showPrintMargin = false;
 
+    /**
+     * @var string Hint to show above the code editor
+     */
+    public $codeHint = '';
```

Comment:
This should either be `hint` that takes a string to render inside of the partial container or probably better you should just render a separate hint field above the code field in the same tab.

Reply:
That's pretty much what I'm doing, just passing the rendered content from the Index controller.

How do you see this exactly?

Reply:
The problem is that you're passing rendered content, that's not done anywhere else in October AFAIK. I'm thinking the cleanest approach is probably to add the hint field as a separate hint field to the same tab that the codeeditor field is in so that it's right next to the codeeditor field but is separate so it requires no additional changes to the API.

Reply:
Ohhh, I see what you mean now. Ok, thanks.